vta/python/vta/exec/rpc_server.py

```diff
@@ -27,7 +27,7 @@ def server_start():
         os.path.abspath(os.path.expanduser(__file__)))
     proj_root = os.path.abspath(os.path.join(curr_path, "../../../../"))
     dll_path = find_libvta()[0]
-    cfg_path = os.path.abspath(os.path.join(proj_root, "build/vta_config.json"))
+    cfg_path = os.path.abspath(os.path.join(curr_path, "../../../config/vta_config.json"))
```

Comment:
So in the tutorial, we instruct to place the vta_config.json file under build for the pynq platform build. Is there a reason to defaulting to the config directory?
Ideally, we'd keep things consistent with the way we look for the json file in `vta/environment.py`: https://github.com/dmlc/tvm/blob/master/vta/python/vta/environment.py#L301-L305
As you can see, it looks in the build directory first. Perhaps we should enforce having only one path to look into rather than multiple ones. It could cause confusing behavior (e.g. editing the wrong config file leads to no effects taken)

Reply:
I agree to have only one path to look into. The reason I assign the `cfg_path` to `vta/config/vta_config.json` is that there already exists a default `vta_config.json` file. If we set `cfg_path` to somewhere else, e.g. `build/vta_config.json`, then I have to manually copy the default configuration file there. In addition, some [other files](https://github.com/dmlc/tvm/blob/master/vta/tests/hardware/pynq/Makefile) in the repository also points to `make/vta_config.json`, which make me very confusing. Therefore, I'm suggesting setting the `cfg_path` to the location where the default config file exists. (I would like to update [the other file](https://github.com/dmlc/tvm/blob/master/vta/tests/hardware/pynq/Makefile) as well.) How is your opinion on this?